commands/retry.js

```diff
@@ -2,6 +2,7 @@
 
 let cli = require('heroku-cli-util')
 let co = require('co')
+let output = require('heroku-apps/lib/output')
```

Comment:
@heroku/cli is this something done in other tools (including another commands dependency)?
Should we extract the output helper somewhere else?